### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by christianbmadsen**
*Issue #, if available:* None

*Description of changes:* Updating the GitHub org readme with more details about Braket and supported repositories.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+* What are [Braket Containers](https://github.com/amazon-braket/amazon-braket-containers) and why do they matter? 
+    * Braket containers are Docker containers that define the runtime environment for an Amazon Braket hybrid job. They allow you to run custom code as part of a co-processing hybrid jobs between a CPU or QPU and a quantum device. AWS [supports a fixed set of containers](https://docs.aws.amazon.com/braket/latest/developerguide/braket-jobs-script-environment.html) and provides regular software updates for feature and security reasons. Minor adjustments to the runtime enviroment can be done by using the Braket base container and adding [a requirements.txt file](https://docs.aws.amazon.com/braket/latest/developerguide/braket-hybrid-job-decorator.html#install-python-packages-and-code). If the runtime environment is very different from what Braket Containters offer you can [bring your own container](https://docs.aws.amazon.com/braket/latest/developerguide/braket-jobs-byoc.html).
```
Comment (virajvchaudhari) [inline profile/README.md:26]: nit: adjustments to the runtime *environment*

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Welcome to the Amazon Braket GitHub organization
+[Amazon Braket](https://aws.amazon.com/braket/) is a fully managed service on Amazon Web Services (AWS) helping researchers, scientists, and developers get started with quantum computing without having to worry setting up cloud infrastructure. 
+
+Quantum computing has the potential to solve computational problems beyond the reach of classical computers because it harnesses the laws of quantum mechanics to process information in new ways. 
+
+A number of opensource projects have been built ontop of Amaozn Braket established. On this page you find starting point to discover these artifacts and how you can contribute to our quantum computing community!
```
Comment (virajvchaudhari) [inline profile/README.md:6]: nit: of *Amazon* Braket established

File: profile/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+* [Braket Python SDK](https://github.com/amazon-braket/amazon-braket-sdk-python) - Is a coding interface for quantum computing devices on Amazon Braket
+* [Braket Examples](https://github.com/amazon-braket/amazon-braket-examples) - Contains tutorials on quantum computing with Amazon Braket
+* [PennyLane-Braket Plugin](https://github.com/amazon-braket/amazon-braket-pennylane-plugin-python) - Enables PennyLane code to run on Amazon Braket
+* [Qiskit-Braket Plugin](https://github.com/qiskit-community/qiskit-braket-provider) - Enables Qiskit code to run on Amazon Braket
+* [Braket Algorithm Library](https://github.com/aws-samples/amazon-braket-algorithm-library/tree/main) - Contains community and first party implementations of quantum algorithms
+* [Braket Containers](https://github.com/amazon-braket/amazon-braket-containers/tree/main) - Offers a set of managed docker containers for hybrid jobs and good starting point for building custom containers
```
Comment (AbeCoull) [inline profile/README.md:16]:
```suggestion
* [Braket Algorithm Library](https://github.com/amazon-braket/amazon-braket-algorithm-library) - Contains community and first party implementations of quantum algorithms
* [Braket Containers](https://github.com/amazon-braket/amazon-braket-containers/tree/main) - Offers a set of managed docker containers for hybrid jobs and good starting point for building custom containers
```

File: profile/README.md (hunk 1 above)
Comment (virajvchaudhari) [reply profile/README.md:26]: very different from what Braket *containers* offer you

File: profile/README.md (hunk 2 above)
Comment (AbeCoull) [inline profile/README.md:6]:
```suggestion
A number of open-source projects have been built ontop of Amaozn Braket established. On this page you find starting point to discover these artifacts and how you can contribute to our quantum computing community! 
```

File: profile/README.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+* [Braket Default Simulators](https://github.com/amazon-braket/amazon-braket-default-simulator-python) - provides an implementation of quantum simulators that you can run locally
+* [Braket Schemas](https://github.com/amazon-braket/amazon-braket-schemas-python) - Used to construct and verify payloads being sent and generated by Amazon Braket
+
+# Frequently Asked Questions (FAQs)
+* Does code written for Braket have to use the Amazon Braket Python SDK?
+  * No, Braket offers simple APIs for a range of programming languages (C++, JavaScript, and more). The Braket service also accepts OpenQASM 3 as the payload for quantum devices. The benefit of integrating with the Amazon Braket Python SDK is that the Python package provides convenience methods for building OpenQASM programs (such as a circuit class) as well as integrations with AWS sessions. We also offer an experimental version of [the Amazon Braket SDK in Julia](https://github.com/awslabs/Braket.jl).
```
Comment (AbeCoull) [inline profile/README.md:22]:
```suggestion
  * No, Braket offers simple APIs for a range of programming languages (C++, JavaScript, and more). The Braket service also accepts OpenQASM 3 as the payload for quantum devices. The benefit of integrating with the Amazon Braket Python SDK is that the Python package provides convenience methods for building OpenQASM programs (such as a circuit class) as well as integrations with AWS sessions. We also offer an experimental version of [the Amazon Braket SDK in Julia](https://github.com/amazon-braket/Braket.jl).
```

**Pull request closed without merging**